tests/config.json

```diff
@@ -13,24 +17,25 @@
             "logName": "s3-recordlog"
         }
     },
-    "kafka": {
```

Comment:
It's odd to see these lines removed in this PR, aren't they needed for some of the tests?

Reply:
Actually talking about `s3` section since kafka has been moved up.

Reply:
hmm, not exactly sure what I did here honestly. Sorry about this, will add back in